graph-db/src/utils/indexing/indexCmrCollection.js

```diff
@@ -0,0 +1,39 @@
+const { indexRelatedUrl } = require('./indexRelatedUrl');
+
+exports.indexCmrCollection = async (collection, gremlin) => {
+    const { meta, umm } = collection;
+    const conceptId = meta['concept-id'];
+    const { EntryTitle, DOI, RelatedUrls } = umm;
```

Comment:
I would suggest dropping in a linter, these TitleCase variables don't match the standard javascript casing and a linter would indicate that for you.

This line would need to change to `const { EntryTitle: entryTitle, DOI: doi, RelatedUrls: relatedUrls } = umm;`